src/tribol/interface/tribol.hpp

```diff
@@ -178,6 +179,62 @@ void registerMesh( integer meshId,
                    const real* y,
                    const real* z=nullptr );
 
+/*!
+ * @brief Define and register a coupling scheme over an MFEM mesh.
+ *
+ * This method is designed to enable simple registration of a contact coupling
+ * scheme over a parallel decomposed MFEM mesh.  Contact surfaces are defined
+ * via a list of boundary attributes, which is used to construct a contact
+ * surface mfem::ParSubMesh.  The parallel domains are then rebalanced using the
+ * redecomp library and the redecomp-level mesh and fields are automatically
+ * registered with Tribol.  Mesh re-decomposition is done with each call to
+ * update(), ensuring the redecomp mesh is still representative of the current
+ * state of the mesh.
+ *
+ * @param [in] cs_id Index to use for the coupling scheme.
+ * @param [in] mesh_id_1 The first ID of the contact surface mesh.
+ * @param [in] mesh_id_2 The second ID of the contact surface mesh.
+ * @param [in] mesh MFEM volume mesh.
+ * @param [in] current_coords Coordinates associated with mesh.
+ * @param [in] attributes_1 Boundary attributes defining the first mesh.
+ * @param [in] attributes_2 Boundary attributes defining the second mesh.
+ * @param [in] contact_mode 
+ * @param [in] contact_case 
+ * @param [in] contact_method 
+ * @param [in] contact_model 
+ * @param [in] enforcement_method 
+ * @param [in] binning_method 
+ */
+void registerMfemMesh( integer cs_id,
+                       integer mesh_id_1,
+                       integer mesh_id_2,
+                       mfem::ParMesh& mesh,
+                       const mfem::ParGridFunction& current_coords,
+                       const std::set<integer>& attributes_1,
```

Comment:
Minor picky comment, but attributes can refer to surface or volume. I realize the comments above clarify this, but maybe b_attributes_1,2 ?

Reply:
Updated.